src/switch_core_media.c

```diff
@@ -10631,59 +10644,62 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_activate_rtp(switch_core_sessi
+							}
+							switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO,
+											  "Activating VIDEO RTCP PORT %d interval %d mux %d\n", remote_port, interval, v_engine->rtcp_mux);
+							switch_rtp_activate_rtcp(v_engine->rtp_session, interval, remote_port, v_engine->rtcp_mux > 0);
 
-					}
+						}
 
 
-					if (v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].ready && v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].ready &&
-						!zstr(v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].con_addr) && 
-						!zstr(v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].con_addr)) {
+						if (v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].ready && v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].ready &&
+							!zstr(v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].con_addr) &&
+							!zstr(v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].con_addr)) {
 
-						if (v_engine->rtcp_mux > 0 && !strcmp(v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].con_addr, v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].con_addr)
-							&& v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].con_port == v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].con_port) {
-							switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO, "Skipping VIDEO RTCP ICE (Same as VIDEO RTP)\n");
-						} else {
+							if (v_engine->rtcp_mux > 0 && !strcmp(v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].con_addr, v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].con_addr)
+								&& v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].con_port == v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].con_port) {
+								switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO, "Skipping VIDEO RTCP ICE (Same as VIDEO RTP)\n");
+							} else {
 
-							switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO, "Activating VIDEO RTCP ICE\n");
-							switch_rtp_activate_ice(v_engine->rtp_session,
-													v_engine->ice_in.ufrag,
-													v_engine->ice_out.ufrag,
-													v_engine->ice_out.pwd,
-													v_engine->ice_in.pwd,
-													IPR_RTCP,
-#ifdef GOOGLE_ICE
-													ICE_GOOGLE_JINGLE,
-													NULL
-#else
-													switch_determine_ice_type(v_engine, session),
-													&v_engine->ice_in
-#endif
-													);
+								switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO, "Activating VIDEO RTCP ICE\n");
+								switch_rtp_activate_ice(v_engine->rtp_session,
+														v_engine->ice_in.ufrag,
+														v_engine->ice_out.ufrag,
+														v_engine->ice_out.pwd,
+														v_engine->ice_in.pwd,
+														IPR_RTCP,
+	#ifdef GOOGLE_ICE
+														ICE_GOOGLE_JINGLE,
+														NULL
+	#else
+														switch_determine_ice_type(v_engine, session),
+														&v_engine->ice_in
+	#endif
+														);
 
 
 
-						}
+							}
 
+						}
```

Comment:
Same here...

This should not be commited. Can you please clean this up @StefanYohansson (maybe you need to checkout the file and re-apply your changes only to relevant part - around lines 4850 - 4880